scripts/rerun_edited_message_logs.py

```diff
@@ -17,6 +18,17 @@ def chat(
     temperature: float = 0.1,
     max_tokens: int = 4096,
 ):
+    # Check if the selected model is available and update if necessary
+    try:
```

Comment:
We don't need this. Also using same AI class?

Reply:
This was requested by @jebarpg so I added it.
I'll remove it.

Reply:
Removed